src/Uno.UI.Runtime.Skia.Wpf/Extensions/UI/Xaml/Controls/TextBoxViewExtension.cs

```diff
@@ -160,7 +164,19 @@ public void SetIsPassword(bool isPassword)
 			// No support for now.
 		}
 
-		public void Select(int start, int length) => _currentInputWidget?.Select(start, length);
+		public void Select(int start, int length)
+		{
+			_currentInputWidget?.Select(start, length);
+
+#if __SKIA__
+			if (_currentInputWidget == null)
+			{
+				this.StartEntry();
+
+				_currentInputWidget!.Select(start, length);
+			}
+#endif
+		}
```

Comment:
I think this would result in the same thing, but is more simple.

```suggestion
		public void Select(int start, int length)
		{
			if (_currentInputWidget == null)
			{
				this.StartEntry();
			}

			_currentInputWidget!.Select(start, length);
		}
```

Reply:
The code block separated by #if __SKIA__ causes additional behavior that just is necessary on SKIA platform to mimic UWP behavior.